### PR TITLE
Links cameraroll library

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: App/Sagas/StorageSagas.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function * refreshLocalImages () {
         const lastRefresh = yield select(StorageSelectors.lastPhotoRefresh)
         // update last time checked to now
         const currentRefresh = (new Date()).getTime()
-        yield call(CameraRoll.requestLocalPhotos, lastRefresh)
+        const results: LocalPhotoResult[] = yield call(CameraRoll.requestLocalPhotos, lastRefresh)
```
Comment (asutula) [inline App/Sagas/StorageSagas.ts]: Cool, `requestLocalPhotos` now works like this. Will keep moving forward with it in the morning.

Comment (andrewxhill) [reply App/Sagas/StorageSagas.ts]: super!

Comment (andrewxhill) [reply App/Sagas/StorageSagas.ts]: "move forward with it" do you intend to do that on this branch, or merge and continue?

Comment (asutula) [reply App/Sagas/StorageSagas.ts]: On this branch

**Comment by asutula** [comment]
Ok, this is G2G and ready for some internal testing. 

Current status:

- Queries camera roll any time the node comes online or app comes back in to foreground
- Adds a Photos menu item for verbose ui that displays the Photos screen
- Photos screen displays photos previously added to local Textile node
- Photos screen displays photos in the process of being added with a little white circle in the lower right corner
- Photos screen display photos that failed to add to the local node with a little red dot in the corner. You can tap the red dot and see a toast of the error message

This  *does not*:

- Remotely pin the photos
- Provide any UI in addition to the thumbnail in the Photos screen

For sure we're going to find some problems with the camera roll querying and the sorting of items in the photos screen, so let's start testing and track down the problems we find.

**Pull request merged**